module1.js

```diff
@@ -23,179 +23,106 @@ function question1(name1) {
   }
 }
 
-
 /**
-  +++++++++++++++++++++++++++++++++++++++++++
   Question 2:
-  IF / ELSE Statements:
-  Check if two numbers are equal, if they are return true or otherwise return false
-  @param {number} number1 a regular integer
-  @param {number} number2 a regular integer
-  @returns {boolean} true or false
-*/
+  Write a function that compares two input integers.
 
-// function question2(number1, number2) {
-//   if (number1) {
-//     // TODO:
-//     return;
-//   } else {
-//     // TODO:
-//     return;
-//   }
-// }
+  Return true if they are equal, or false otherwise.
+*/
 
+function question2(number1, number2) {
+  // TODO:
+}
 
 /**
-  +++++++++++++++++++++++++++++++++++++++++++
   Question 3:
-  Is the number even or odd?
-  @param {number} number1 a regular integer
-  @returns {string} odd or even
-*/
+  Write a function that checks if an input integer is an even or an odd number.
 
-// function question3(number1) {
-//   if (number1) {
-//     return;
-//   }
-// }
+  Return a string "odd" if the number is odd, or a string "even" if the number is even.
+*/
 
+function question3(number) {
+  // TODO:
+}
 
 /**
-  +++++++++++++++++++++++++++++++++++++++++++
   Question 4:
-  Write a for loop that loops over the numbers from 0 to 100, returning an array containing only every second number. So it should return [ 0, 2, 4, 6, 8, 10 .... etc.]
+  Write a function that takes an array of integers as input, and returns an array containing only every second element.
 
-  @param none
-  @returns [array 0 ... 100] 
+  Use the .forEach() or .map() functions to do this.
+  
+  For example, if the input array is: [1,2,3,4,5,6]
+  the output would be: [2,4,6]
 */
 
-// function question4() {
-//   // TODO
-//   for ( /* ??? */ ) {
-//     // TODO
-//   }
-//   return /* ??? */;
-// }
-
+function question4(array) {
+  // TODO:
+}
 
 /**
-  +++++++++++++++++++++++++++++++++++++++++++
   Question 5:
-  Write a function called multiply that multiplies two numbers together. It should take two numbers as parameters and return the answer.
+  Write a function that multiplies two input integers or two input floats (positive or negative).
```

Comment:
I think it might be confusing to mention integers and floats. In terms of this question there's no real difference

Reply:
Done.
I also realised that the test tries to multiply `0.7` and `2.3`, if you just do `0.7 * 2.3` the result is 1.6099999 - and the test expects `1.61` - do you think that could be a problem?

Reply:
Yay for JavaScript floats! 🎉 I can only imagine that was intentional to test for knowledge about this. However I don't think we covered that in the syllabus (you probably know better than I do), so I'd be inclined to remove it. I don't necessarily think it's core to JS knowledge either.